### PR TITLE
chore: add comment to header sync with empty

### PR DESCRIPTION
**Pull request by driftluo**
### What problem does this PR solve?

add comment to header sync with empty

### Check List

Tests

- Unit test
- Integration test

### Release note
```release-note
None: Exclude this PR from the release note.
```

### CODE REVIEW

File: sync/src/synchronizer/block_fetcher.rs [hunk 1]
```diff
@@ -98,7 +98,16 @@ impl<'a> BlockFetcher<'a> {
         }
 
         // This peer has nothing interesting.
```
Comment (eval-exec) [inline sync/src/synchronizer/block_fetcher.rs:100]: I think this comment is unrelated to the code below.

**Review by eval-exec — APPROVED** [review]

**Review by eval-exec — APPROVED** [review]

**Review by zhangsoledad — APPROVED** [review]
